### PR TITLE
[zh] sync and update fine-parallel-processing-work-queue

### PR DESCRIPTION
**Pull request by yangjiao2**
1. change keyword replicationController to ReplicaSet
2. change word 连续 to 持续
3. other minor changes (e.g: add connection word for readability, add 'shell' for command line input)

### CODE REVIEW

**Comment by yangjiao2** [comment]
I have got CNCF signed with my Gitlab account and I wanner how could I pass that check?
Want to contribute on this amazing doc! :p 
Thanks 

image

**Comment by yangjiao2** [comment]
I signed it

**Comment by tengqm** [comment]
@yangjiao2 You may want to check if you committed the PR using the same email account for CLA. You may need to recommit the change if necessary.

**Comment by yangjiao2** [comment]
Thanks @tengqm  will try in a few minutes and see how that goes

File: content/zh/docs/tasks/job/fine-parallel-processing-work-queue.md [hunk 1]
```diff
@@ -364,7 +364,6 @@ consider running your background workers with a `replicationController` instead,
 and consider running a background processing library such as
 [https://github.com/resque/resque](https://github.com/resque/resque).
 -->
-如果你有连续的后台处理业务，那么可以考虑使用 `replicationController` 来运行你的后台业务，
+如果你有持续的后台处理业务，那么可以考虑使用 `ReplicaSet` 来运行你的后台业务，
```
Comment (tengqm) [inline content/zh/docs/tasks/job/fine-parallel-processing-work-queue.md:367]: 英文原文是 "replicationController"，这个fix合法么？

**Pull request merged**